common/featureset.cpp

```diff
@@ -20,6 +20,7 @@ void FeatureSet::initalizeFeatureList(QMap<QString, bool> &featureList) {
     featureList.insert("room_chat_history", false);
     featureList.insert("client_warnings", false);
     featureList.insert("mod_log_lookup", false);
+    featureList.insert("client_inactivetimeout", false);
```

Comment:
👎  this feature isn't related to client-server communication so there's no need for the server to know about it.